### PR TITLE
Add background color support

### PR DESCRIPTION
**Pull request by f3lin3**
Does this pass as a first attempt at achieving the requirements of https://github.com/ififfy/flipflip/issues/86 ?

### CODE REVIEW

**Comment by f3lin3** [comment]
I'm still getting to grips with github electron yarn et al.  I am not sure where my pull request explanation went... :)

The additions permit a color name (e.g. "black" or "red" etc...) or a color number (e.g. #ff00ff) to be entered into the Background Color text control in Scene Detail.  - If such is entered then that background is used.

If the Background Color is left empty then the default blurred and scaled background image is used.

File: src/renderer/components/player/ImagePlayer.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import fs from "fs";
 import gifInfo from 'gif-info';
 import ChildCallbackHack from './ChildCallbackHack';
 import urlToPath from '../../urlToPath';
+import { isNull, isNullOrUndefined } from 'util';
```
Comment (ififfy) [inline src/renderer/components/player/ImagePlayer.tsx]: These aren't used, please remove them

**Comment by ififfy** [comment]
Looks good to me! I'll merge once you remove the unused import.

File: src/renderer/components/player/ImagePlayer.tsx (hunk 1 above)
Comment (f3lin3) [reply src/renderer/components/player/ImagePlayer.tsx]: - Spurious unused import has now been removed (not sure where that artefact came from)

- Additions post original pull request
I have added a Simple Color Picker (SimpleColorPicker.tsx)  - I am not sure if the approach I took to the isVisible property and its use in the render method is the approved way of doing things - but the effect in the Scene Details is to only display the SimpleColorPicker when the user chooses a fixed color for the background - when the choice is a blurred image the SimpleColorPicker is removed.   There may be cleaner methods of achieving this - let me known and I'll have a go at remediation.

**Comment by ififfy** [comment]
Can you attach a screenshot of the color picker?

File: src/renderer/components/ui/SimpleColorPicker.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    label: string,
+    value: string,
+    isVisible: boolean,
+    isEnabled: boolean,
+    children?: React.ReactNode,
+    onChange: (value: string) => void
+  };
+
+    render() {
+        if (this.props.isVisible) {
```
Comment (ififfy) [inline src/renderer/components/ui/SimpleColorPicker.tsx]: Please don't put `isVisible` in this component. Use `this.props.scene.backgroundType== BT.color && <SimpleColorPicker...` instead where the component is used.

Comment (f3lin3) [reply src/renderer/components/ui/SimpleColorPicker.tsx]: Ok give me an hour or so

**Comment by f3lin3** [comment]
`isVisible` has been removed, and I think I have updated the `render()` function in the EffectGroups.tsx appropriately.

**Comment by f3lin3** [comment]
> Can you attach a screenshot of the color picker?

![screenshot from 2019-01-27 20-28-15](https://user-images.githubusercontent.com/46749380/51806535-c815d180-2272-11e9-8c08-8862911db6f7.png

**Comment by f3lin3** [comment]
It might be easier to reject this pull request - in favor of #88  - as it squashes the changes into one commit?  (I am a git noob :))

**Comment by GoddessServant0** [comment]
Closing in favor of #88

**Pull request closed without merging**